lms/static/js/jquery.gradebook.js

```diff
@@ -57,16 +58,23 @@ var Gradebook = function($element) {
 	};
 
 	var updateWidths = function(e) {
+		// Here updating direction of scroll.
 		maxScroll = $gradeTable.width() - $grades.width();
-		var targetLeft = clamp($gradeTable.position().left, -maxScroll, 0);
+		var targetLeft = clamp($gradeTable.position().left, maxScroll, 0);
 		updateHorizontalPosition(targetLeft);
 		setShadows(targetLeft);
 	};
 
+	/**
+	 * On dragging we compute pixels on the base of how mouse movement.
+	 * Then we are using those pixels to scroll left or right.
+	 * (Note: In past grade book was using dragging to
+	 * view hidden columns on viewport).
```

Comment:
Comments that reference what the code *used* to do are confusing, and should be removed. Comments should always reflect the *current* state of the code.